pkg/util/buildkitutil/buildkit.go

```diff
@@ -20,21 +20,31 @@ import (
 )
 
 const buildkitConfigTemplate = `
-[registry."{{ if .Registry }}{{ .Registry }}{{ else }}docker.io{{ end }}"]{{ if .Mirror }}
-  mirrors = ["{{ .Mirror }}"]{{ end }}
-  http = {{ .UseHTTP }}
-  {{ if .SetCA}}ca=["/etc/registry/ca.pem"]
-  [[registry."{{ if .Registry }}{{ .Registry }}{{ else }}docker.io{{ end }}".keypair]]
-	key="/etc/registry/key.pem"
-	cert="/etc/registry/cert.pem"
-  {{ end }}
+[registry]
+{{- range $index, $value := .Registry }}
+  [registry."{{ if $value }}{{ $value }}{{ else }}docker.io{{ end }}"]
+  	{{- if $.UseHTTP }}
+    http = true
```

Comment:
what if they want to specify different http/mirrors for different registries?

Reply:
Maybe we should make the mirror a string slice.

Reply:
Sure I'll make that change.

Reply:
cc @kemingy WDYT